techniques/html/H44.html

```diff
@@ -136,64 +117,56 @@ <h2>Description</h2>
+                     <li>スクリプトボタン (<code class="el">button</code> elements or &lt;<code class="el">input</code>
                         										<code class="att">type</code>="button"&gt;)
                      </li>
                      
                   </ul>
-                  
+                  <div class="note"><div role="heading" class="note-title marker" aria-level="3">訳注:</div><div><p>WAIC では H44 に関するアクセシビリティ サポーテッド（AS）情報を提供している。2020 年 3 月版の アクセシビリティ サポーテッド（AS）情報: H44 も参照されたい。</p></div></div>
                </div>
             </div>
          </section>
          <section id="examples">
-            <h2>Examples</h2>
+            <h2>事例</h2>
             <section class="example" id="example-1-a-text-input-field">
-               <h3>Example 1: A text input field</h3>
-               <p>The text field in the example below has the explicit label of "First name:". The
-                  <code class="el">label</code> element's <code class="att">for</code> attribute matches the
-                  <code class="att">id</code> attribute of the <code class="el">input</code> element. 
+               <h3>事例 1: テキスト入力フィールド</h3>
+               <p>この事例では、テキスト入力フィールドに「First name:」という明示的なラベルが付けてある。<code class="el">label</code> 要素の for 属性値は、<code class="el">input</code> 要素の <code class="att">id</code> 属性値と一致している。
                </p><pre xml:space="preserve">&lt;label for="firstname"&gt;First name:&lt;/label&gt; 
 &lt;input type="text" name="firstname" id="firstname" /&gt;</pre></section>
             <section class="example" id="example-2-a-checkbox">
-               <h3>Example 2: A checkbox</h3><pre xml:space="preserve">&lt;input type="checkbox" id="markuplang" name="computerskills" checked="checked"&gt;
+               <h3>事例 2: チェックボックス</h3><pre xml:space="preserve">&lt;input type="checkbox" id="markuplang" name="computerskills" checked="checked"&gt;
 &lt;label for="markuplang"&gt;HTML&lt;/label&gt;</pre></section>
             <section class="example" id="example-3-a-group-of-radio-buttons">
-               <h3>Example 3: A group of radio buttons</h3>
-               <p>A small, related group of radio buttons with a clear description and labels for
-                  each individual element.
+               <h3>事例 3: ラジオボタンのグループ</h3>
+               <p>関連するラジオボタンの小さなグループについて、簡単な説明が付けてあり、さらに個々の要素にラベルが付けてある。
                </p>
                <div class="note">
-                  <div role="heading" class="note-title marker" aria-level="4">Note</div>
+                  <div role="heading" class="note-title marker" aria-level="4">注記</div>
                   <div>
                      
-                     <p>To provide clear associations and instructions for a large set of related radio
-                        buttons <a href="#H71"></a>, should be considered.
+                     <p>ラジオボタン<a href="#H71"></a>の数が多く、それらの関連付けや操作説明を必要とする場合は、考慮すべきである。
```

Comment:
これは完全に原文が悪いのですが、空のリンクを埋めておきます。
```suggestion
                     <p>ラジオボタンの数が多く、それらの関連付けや操作説明を必要とする場合は、<a href="#H71">H71: fieldset 要素及び legend 要素を使用して、フォームコントロールのグループに関する説明を提供する</a>の方法を考慮すべきである。
```